### PR TITLE
Enhancing the design 

### LINKED ISSUE #89: Enhancing the design of the home page
**Task title**
 Enhancing the design of the home page 

**Task description**
We aim to enhance the design of our home page to provide a more appealing and user-friendly experience. 

**Subtasks**

- [x] enhance the home page design
- [x] add animation 
- [x] #98
- [x] #99

### PR DESCRIPTION
**Pull request by Lilicod**
### Changes 

- Enhancing the design of the home page
- Add animation on scroll for the home page
- add animation on scroll to the navbar
- Add profile icon on the navbar and a user dropdown
- Enhance the design for all the pages

closes #89 

# Related Issue

- Resolve #89

### CODE REVIEW

**Review by M07am3dM — APPROVED** [review]
Nice work @Lilicod well done, however still few things need to be fixed at last, thank you.

File: public/Images/1.jpg
Comment (dzc0d3r) [inline public/Images/1.jpg:1]: 5MB is a big size .. we need to address this

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
         "date-fns": "^2.29.1",
         "eslint-plugin-jest": "^26.6.0",
         "firebase": "^10.4.0",
+        "framer-motion": "^10.16.4",
```
Comment (dzc0d3r) [inline package.json:30]: Awesome thank you @Lilicod

File: public/Images/30.jpg
Comment (dzc0d3r) [inline public/Images/30.jpg:1]: Almost 8MB, too much

File: public/Images/therapist.jpg
Comment (dzc0d3r) [inline public/Images/therapist.jpg:1]: this one too

File: src/layout/Layout.jsx [hunk 2]
```diff
@@ -5,18 +5,23 @@ import "react-toastify/dist/ReactToastify.css";
...
+            <motion.div
+                initial={{ opacity: 0, y: 15 }}
+                animate={{ opacity: 1, y: 0 }}
+                exit={{ opacity: 0, y: 15 }}
+                transition={{ delay: 0.25 }}
+            >
+                {children}
+            </motion.div>
             <ToastContainer position='top-center' />
             {children}
```
Comment (dzc0d3r) [inline src/layout/Layout.jsx]: I was wondering why it shows the children two time on each other  page .. we need to remove this extra `{children}`

**Review by dzc0d3r — APPROVED** [review]
Great Job @Lilicod , 
However i want to mention few things to keep track for later: 
-  Some of the images are too big
-  The layout renders the children two times. 
-  There is some few problems with responsiveness here and there.

We need to address these issues later.

Overall you did an Amazing job, thank you @Lilicod  :1st_place_medal:

File: src/layout/Layout.jsx (hunk 2 above)
Comment (Lilicod) [reply src/layout/Layout.jsx]: it's because the develop was merged to this branch I'm working on fixing it

**Comment by Lilicod** [comment]
> Great Job @Lilicod , However i want to mention few things to keep track for later:
> 
> * Some of the images are too big
> * The layout renders the children two times.
> * There is some few problems with responsiveness here and there.
> 
> We need to address these issues later.
> 
> Overall you did an Amazing job, thank you @Lilicod 🥇

thank you @dzc0d3r for the layout issue I'll fix it. about the images I noticed that too, I'll try to find a solution for it

**Pull request merged**